calcom/cal.com
# Adds number for sms notifications from workflows to /success

## What does this PR do?

- Show SMS reminder number on /success page with all other booking details
- Change SMS reminder to SMS notification (it does not need to be a reminder)

![Screenshot 2022-10-25 at 17 54 37](https://user-images.githubusercontent.com/30310907/197822515-e29a755b-718b-47e1-a578-526e40714d1f.png)

![Screenshot 2022-10-25 at 17 49 46](https://user-images.githubusercontent.com/30310907/197822374-36b0d55a-7427-43a8-b73e-e7c7a2368510.png)


Fixes #4813 (https://github.com/calcom/cal.com/issues/4813#issuecomment-1282200040)

**Environment**: Staging(main branch)


## How should this be tested?

- Create a workflow and add 'send SMS to attendee' as an action
- Set workflow active on one event type and book this event type
- See that /success shows the phone number for SMS notifications
